src/sagemaker/deserializers.py

```diff
@@ -60,3 +62,34 @@ def deserialize(self, data, content_type):
             return data.read()
         finally:
             data.close()
+
+
+class CSVDeserializer(BaseDeserializer):
+    """Deserialize a stream of bytes into a list of lists."""
+
+    ACCEPT = "test/csv"
+
+    def __init__(self, encoding="UTF-8"):
+        """Initialize the string encoding.
+
+        Args:
+            encoding (str): The string encoding to use (default: "UTF-8").
+        """
+        self.encoding = encoding
+
+    def deserialize(self, data, content_type):
+        """Deserialize data from an inference endpoint into a list of lists.
+
+        Args:
+            data (object): Data to be deserialized.
+            content_type (str): The MIME type of the data.
+
+        Returns:
+            list: The data deserialized into a list of lists representing the
+                contents of a CSV file.
+        """
+        try:
+            string = data.read().decode(self.encoding)
```

Comment:
You should probably use a different name, since `string` is the name of a [module in the standard library](https://docs.python.org/3/library/string.html).

Reply:
Fixed